src/System.Windows.Forms/src/System/Windows/Forms/AccessibleNavigation.cs

```diff
@@ -2,107 +2,54 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace System.Windows.Forms {
-
-    using System.Diagnostics;
-
-    using System;
-    using System.ComponentModel;
-    using Microsoft.Win32;
-
-
-    /// <include file='doc\AccessibleNavigation.uex' path='docs/doc[@for="AccessibleNavigation"]/*' />
+namespace System.Windows.Forms
+{
     /// <devdoc>
-    ///    <para>
-    ///       Specifies values for navigating between accessible objects.
-    ///    </para>
+    /// Specifies values for navigating between accessible objects.
     /// </devdoc>
-    public enum AccessibleNavigation {
-
-        /// <include file='doc\AccessibleNavigation.uex' path='docs/doc[@for="AccessibleNavigation.Down"]/*' />
+    public enum AccessibleNavigation
+    {
         /// <devdoc>
-        ///    <para>
-        ///       Navigation
-        ///       to a sibling object located
-        ///       below the
-        ///       starting object.
-        ///    </para>
+        /// Navigation to a sibling object located below the starting object.
         /// </devdoc>
         Down = 0x2,
 
-        /// <include file='doc\AccessibleNavigation.uex' path='docs/doc[@for="AccessibleNavigation.FirstChild"]/*' />
         /// <devdoc>
-        ///    <para>
-        ///       Navigation to
-        ///       the
-        ///       first child of the object.
-        ///    </para>
+        /// Navigation to the first child of the object.
         /// </devdoc>
         FirstChild = 0x7,
 
-        /// <include file='doc\AccessibleNavigation.uex' path='docs/doc[@for="AccessibleNavigation.LastChild"]/*' />
         /// <devdoc>
-        ///    <para>
-        ///       Navigation to
-        ///       the
-        ///       last child of the object
-        ///    </para>
+        /// Navigation to the last child of the object
         /// </devdoc>
         LastChild = 0x8,
 
-        /// <include file='doc\AccessibleNavigation.uex' path='docs/doc[@for="AccessibleNavigation.Left"]/*' />
         /// <devdoc>
-        ///    <para>
-        ///       Navigation
-        ///       to the sibling object located
-        ///       to the left
-        ///       of the
-        ///       starting object.
-        ///    </para>
+        /// Navigation to the sibling object located to the left of the
+        /// starting object.
         /// </devdoc>
         Left = 0x3,
 
-        /// <include file='doc\AccessibleNavigation.uex' path='docs/doc[@for="AccessibleNavigation.Next"]/*' />
         /// <devdoc>
-        ///    <para>
-        ///       Navigation
-        ///       to the next logical object, generally from a
-        ///       sibling
-        ///       object to the
-        ///       starting object.
-        ///    </para>
+        /// Navigation to the next logical object, generally from a sibling
```

Comment:
It seems that the comment can be corrected here to have
"Navigation to the next logical object, generally **from the starting object to a sibling object.**"

Unfortunately in docs there is no explicit point to the direction:
https://docs.microsoft.com/ru-ru/windows/desktop/winauto/navigation-constants
NAVDIR_NEXT | Navigate to the next logical object; generally, it is a sibling of the starting object.
NAVDIR_PREVIOUS | Navigate to the previous logical object; generally, it is a sibling of the starting object.

```suggestion
        /// Navigation to the next logical object, generally from the starting
```